libs/perun/dialogs/src/lib/applications-bulk-operation-dialog/applications-bulk-operation-dialog.component.html

```diff
@@ -0,0 +1,63 @@
+<div class="{{data.theme}}">
+  <h1 mat-dialog-title>{{ data.title | translate }}</h1>
+  <div mat-dialog-content>
+    <div *ngIf="data.action==='RESEND'">
+      {{'DIALOGS.RE_SEND_NOTIFICATION.SELECT' | translate}}:
+      <mat-form-field class="w-100">
+        <mat-select [(value)]="mailType" disableOptionCentering>
+          <mat-option value="APP_CREATED_USER">
+            {{'DIALOGS.RE_SEND_NOTIFICATION.NOTIFICATION_TYPE.APP_CREATED_USER' | translate}}
+          </mat-option>
+          <mat-option *ngIf="data.allowGroupMailType" value="APPROVABLE_GROUP_APP_USER">
+            {{'DIALOGS.RE_SEND_NOTIFICATION.NOTIFICATION_TYPE.APPROVABLE_GROUP_APP_USER' | translate}}
+          </mat-option>
+          <mat-option value="APP_CREATED_VO_ADMIN">
+            {{'DIALOGS.RE_SEND_NOTIFICATION.NOTIFICATION_TYPE.APP_CREATED_VO_ADMIN' | translate}}
+          </mat-option>
+          <mat-option value="MAIL_VALIDATION">
+            {{'DIALOGS.RE_SEND_NOTIFICATION.NOTIFICATION_TYPE.MAIL_VALIDATION' | translate}}
+          </mat-option>
+          <mat-option value="APP_APPROVED_USER">
+            {{'DIALOGS.RE_SEND_NOTIFICATION.NOTIFICATION_TYPE.APP_APPROVED_USER' | translate}}
+          </mat-option>
+          <mat-option value="APP_REJECTED_USER">
+            {{'DIALOGS.RE_SEND_NOTIFICATION.NOTIFICATION_TYPE.APP_REJECTED_USER' | translate}}
+          </mat-option>
+          <mat-option value="APP_ERROR_VO_ADMIN">
+            {{'DIALOGS.RE_SEND_NOTIFICATION.NOTIFICATION_TYPE.APP_ERROR_VO_ADMIN' | translate}}
+          </mat-option>
+        </mat-select>
+      </mat-form-field>
+
+      <div *ngIf="mailType === 'APP_REJECTED_USER'">
+        {{'DIALOGS.RE_SEND_NOTIFICATION.REASON' | translate}}:
+        <mat-form-field class="w-100">
+          <textarea [(ngModel)]="reason" matInput></textarea>
+        </mat-form-field>
+      </div>
+    </div>
+
+    <perun-web-apps-alert alert_type="warn">
+      {{ data.description | translate: {count: dataSource.data.length} }}
+    </perun-web-apps-alert>
+
+    <div class="table">
+      <mat-table [dataSource]="dataSource" matSort>
```

Comment:
This table only displays data for the default columns (see image).
![Screenshot from 2023-09-14 09-41-58](https://github.com/CESNET/perun-web-apps/assets/97669007/3a31c4cb-e15a-4dfa-bc2f-c1a37c8df3df)
At least group info should work IMO, the rest can be omitted.